server.js

```diff
@@ -51,6 +59,26 @@ function searchLocationData(request, response) {
   //user input - ex: if they type in Seattle...search_query = Seattle
   const search_query = request.query.data;
   const URL = `https://maps.googleapis.com/maps/api/geocode/json?address=${search_query}&key=${process.env.GEOCODE_API_KEY}`;
+  client.query(SQL.getLocation, [search_query].then(result => {

```

Comment:
There is a syntax error on this line. A missing end parenthesis. And there is an extra end parenthesis further down the page.